### PR TITLE
Added Gate facade

### PR DESCRIPTION
**Pull request by hassankhan**
Added `Gate` facade to documentation

### CODE REVIEW

File: facades.md [hunk 1]
```diff
@@ -78,6 +78,7 @@ DB  |  [Illuminate\Database\DatabaseManager](http://laravel.com/api/{{version}}/
 DB (Instance)  |  [Illuminate\Database\Connection](http://laravel.com/api/{{version}}/Illuminate/Database/Connection.html)  |
 Event  |  [Illuminate\Events\Dispatcher](http://laravel.com/api/{{version}}/Illuminate/Events/Dispatcher.html)  |  `events`
 File  |  [Illuminate\Filesystem\Filesystem](http://laravel.com/api/{{version}}/Illuminate/Filesystem/Filesystem.html)  |  `files`
++Gate  |  [Illuminate\Auth\Access\Gate](http://laravel.com/api/{{version}}/Illuminate\Auth\Access\Gate)  |  `gate`
```
Comment (GrahamCampbell) [inline facades.md]: this looks wrong to me - doesn't match every other link in this file...

**Comment by taylorotwell** [comment]
Yeah the slashes are wrong.

File: facades.md (hunk 1 above)
Comment (GrahamCampbell) [inline facades.md]: also, why is there a ++?

Comment (GrahamCampbell) [inline facades.md]: Illuminate\Auth\Access\Gate is also wrong anyway

Comment (GrahamCampbell) [inline facades.md]: should be the contract

Comment (GrahamCampbell) [inline facades.md]: and that "gate" thing is wrong too - no such binding exists

**Comment by GrahamCampbell** [comment]
See https://github.com/laravel/docs/pull/1782.

**Comment by GrahamCampbell** [comment]
Everything in this PR was wrong I'm afraid, so I redid it.

**Comment by hassankhan** [comment]
I'm sorry, was in a hurry to leave the office, my bad.

**Pull request closed without merging**